wagtail/admin/userbar.py

```diff
@@ -84,7 +86,9 @@ class AccessibilityItem(BaseItem):
     axe_custom_checks = [
         {
             "id": "check-image-alt-text",
-            "options": {"pattern": "\\.(avif|gif|jpg|jpeg|png|svg|webp)$|_"},
+            "options": {
+                "pattern": f"\\.({'|'.join(get_allowed_image_extensions())})$|_"
```

Comment:
I think the problem is if the `get_allowed_image_extensions()` returns a _subset_ of all the formats that Wagtail can support.

Example:

- Set `WAGTAILIMAGES_EXTENSIONS` to `["avif", "jpg", "png"]`
- Upload an `.avif` image with the filename as the alt text
- Set `WAGTAILIMAGES_EXTENSIONS` to `["jpg", "png"]`

If the image is used on the page, the checker will not flag the alt text, because it's not in `WAGTAILIMAGES_EXTENSIONS`.

Reply:
True. What if we checked against _all_ valid image file types using Django's equivalent method: https://github.com/django/django/blob/163e72ebbaa84804877f3d1ae212575e479b533b/django/core/validators.py#L660 - would that be better?